### DISCUSSION TITLE
Adding contacts from the Spreadsheet to contacts en masse

### DISCUSSION THREAD

**Discussion by yusufsunetci** [FossifyOrg/General-Discussion#535] [Q&A]
There are about 200 people in LibreOffice Calc. I want to add these people to the contacts without deleting the ones in my contacts. How can I do this?

**Comment by Aga-C** [comment]
1. Fossify Phone doesn't handle adding contacts. Fossify Contacts is responsible for this.
2. Fossify Contacts supports importing contacts only from VCF files.
